jwst/datamodels/schemas/cube.schema.yaml

```diff
@@ -32,8 +32,6 @@ allOf:
       default: 0.0
       ndim: 2
       datatype: float32
-    relsens:
-      $ref: relsens.schema.yaml
```

Comment:
Should you delete the relsens.schema.yaml file?

Reply:
Done it latest commit.